src/views/Correction.vue

```diff
@@ -1,210 +1,46 @@
 <template>
-  <section id="correction-view">
-    <header>
-      <h1>Correction - Incorporation Application</h1>
-    </header>
-
-    <section id="original-filing-date" class="mt-6">
-      <p>
-        <span id="original-filing-date-label">Original Filing Date:</span>
-        {{ originalFilingDate }}
-      </p>
-    </section>
-
-    <section id="benefit-company-statement" class="mt-6" v-if="isTypeBcomp">
-      <p>
-        <span id="benefit-company-statement-label">{{ BenefitCompanyStatementResource.title }}:</span>
-        {{ BenefitCompanyStatementResource.description }}
-      </p>
-    </section>
-
-    <YourCompany class="mt-10" />
-
-    <PeopleAndRoles class="mt-10" />
-
-    <ShareStructures class="mt-10" />
-
-    <AgreementType class="mt-10" />
-
-    <CompletingParty class="mt-10" sectionNumber="1." />
-
-    <Detail class="mt-10" sectionNumber="2." validate="true" />
-
-    <CertifySection class="mt-10" sectionNumber="3." validate="true" />
-
-    <StaffPayment
-      class="mt-10"
-      sectionNumber="4."
-      @haveChanges="onStaffPaymentChanges()"
-    />
-  </section>
+  <div>
+    <template v-if="isTypeBcomp || true">
+      <BenCorrection
+        :appReady="appReady"
+        @fetchError="emitFetchError($event)"
+        @haveData="emitHaveData($event)"
+      />
+    </template>
+    <template v-if="isTypeFirm">
+      <FmCorrection
+        :appReady="appReady"
+        @fetchError="emitFetchError($event)"
+        @haveData="emitHaveData($event)"
+      />
+    </template>
```

Comment:
This "general correction" page now displays one of the specific correction pages.